### PR TITLE
Modifies the parent README.md of the repository

### PR DESCRIPTION
**Pull request by sachithKay**

### CODE REVIEW

**Comment by Nirothipan** [comment]
Shall we update https://github.com/wso2/product-mi-tooling/blob/master/monitoring-dashboard/README.md as well with new folder structure

**Comment by sachithKay** [comment]
> Shall we update https://github.com/wso2/product-mi-tooling/blob/master/monitoring-dashboard/README.md as well with new folder structure

+1. README.md files for both CLI and dashboard has been changed accordingly

File: README.md [hunk 1]
```diff
@@ -3,12 +3,22 @@
...
+
+# product-mi-components
+Two main tooling artifacts included here are as follows.
+1. [Command Line Interface](https://github.com/wso2/product-mi-tooling/tree/master/cmd) for WSO2 Micro Integrator.
+2. [Monitoring Dashboard](https://github.com/wso2/product-mi-tooling/tree/master/monitoring-dashboard) for WSO2 Micro Integrator.
+
+* Please refer the README.md files relevant to each module for more information on each of the modules.
+
+
+#How to Contribute
```
Comment (Nirothipan) [inline README.md]:
```suggestion
##  How to Contribute
```

File: README.md [hunk 2]
```diff
@@ -3,12 +3,22 @@
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/product-mi-tooling)](https://wso2.org/jenkins/job/products/job/product-mi-tooling/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-This repository houses tooling components required for WSO2 Micro Integrator. Two main tooling artifacts included 
-here are as follows.
-1. Command Line Interface for WSO2 Micro Integrator
-2. Monitoring Dashboard for WSO2 Micro Integrator
+This repository houses tooling components required for WSO2 Micro Integrator. Each of these components interact with
```
Comment (nilminiwso2) [inline README.md]: This repository stores the tooling components required for WSO2 Micro Integrator. Each of these components interact with 
the Management API of WSO2 Micro Integrator to provide the information related to a specified server instance.

File: README.md [hunk 3]
```diff
@@ -3,12 +3,22 @@
...
-2. Monitoring Dashboard for WSO2 Micro Integrator
+This repository houses tooling components required for WSO2 Micro Integrator. Each of these components interact with 
+the Management API of the WSO2 Micro Integrator to provide the information related to a specified server instance. 
+
+# product-mi-components
+Two main tooling artifacts included here are as follows.
+1. [Command Line Interface](https://github.com/wso2/product-mi-tooling/tree/master/cmd) for WSO2 Micro Integrator.
+2. [Monitoring Dashboard](https://github.com/wso2/product-mi-tooling/tree/master/monitoring-dashboard) for WSO2 Micro Integrator.
+
+* Please refer the README.md files relevant to each module for more information on each of the modules.
```
Comment (Nirothipan) [inline README.md]: Shall we update this as, for more information on each component
1) Command Line Interface
2) Monitoring Dashboard

and link the readme md files

File: README.md [hunk 4]
```diff
@@ -3,12 +3,22 @@
...
 
-This repository houses tooling components required for WSO2 Micro Integrator. Two main tooling artifacts included 
-here are as follows.
-1. Command Line Interface for WSO2 Micro Integrator
-2. Monitoring Dashboard for WSO2 Micro Integrator
+This repository houses tooling components required for WSO2 Micro Integrator. Each of these components interact with 
+the Management API of the WSO2 Micro Integrator to provide the information related to a specified server instance. 
+
+# product-mi-components
+Two main tooling artifacts included here are as follows.
```
Comment (nilminiwso2) [inline README.md]: The two main tooling artifacts included here are as follows:

File: cmd/README.md [hunk 5]
```diff
@@ -8,13 +8,13 @@ Command Line tool for managing the WSO2 Micro Integrator
...
     3. Clone your fork into any directory
-    5. `cd` into cloned directory and then cd into `micro-integrator/cmd`
+    5. Navigate into cloned directory and then navigate into `product-mi-tooling/cmd`
     6. Execute `go mod vendor` or `go mod download` to download all the dependencies
     
 - ### Building from source
-    cd into micro-integrator home directory
+    Navigate into product-mi-tooling home directory
 
     Execute `make install` to build both the **Micro Integrator** and **MIcro Integrator CLI** at once.
```
Comment (Nirothipan) [inline cmd/README.md]: This also has to be updated as we have only CLI now .

File: README.md [hunk 6]
```diff
@@ -3,12 +3,22 @@
...
+1. [Command Line Interface](https://github.com/wso2/product-mi-tooling/tree/master/cmd) for WSO2 Micro Integrator.
+2. [Monitoring Dashboard](https://github.com/wso2/product-mi-tooling/tree/master/monitoring-dashboard) for WSO2 Micro Integrator.
+
+* Please refer the README.md files relevant to each module for more information on each of the modules.
+
+
+#How to Contribute
+- Please report issues at [Github](https://github.com/wso2/micro-integrator).
```
Comment (nilminiwso2) [inline README.md]: on

File: cmd/README.md [hunk 7]
```diff
@@ -8,13 +8,13 @@ Command Line tool for managing the WSO2 Micro Integrator
 
 - ### Setting up the development environment
     1. Install [Go 1.12.x](https://golang.org/dl)
-    2. Fork the [repository](https://github.com/wso2/micro-integrator)
+    2. Fork the [repository](https://github.com/wso2/product-mi-tooling)
     3. Clone your fork into any directory
-    5. `cd` into cloned directory and then cd into `micro-integrator/cmd`
+    5. Navigate into cloned directory and then navigate into `product-mi-tooling/cmd`
```
Comment (nilminiwso2) [inline cmd/README.md]: Navigate to the cloned directory and then to the `product-mi-tooling/cmd` directory

File: README.md (hunk 3 above)
Comment (sachithKay) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (sachithKay) [reply README.md]: Fixed

File: cmd/README.md (hunk 5 above)
Comment (sachithKay) [reply cmd/README.md]: Fixed

File: README.md (hunk 6 above)
Comment (sachithKay) [reply README.md]: fixed

File: cmd/README.md (hunk 7 above)
Comment (sachithKay) [reply cmd/README.md]: fixed

File: cmd/README.md [hunk 8]
```diff
@@ -8,13 +8,13 @@ Command Line tool for managing the WSO2 Micro Integrator
...
     6. Execute `go mod vendor` or `go mod download` to download all the dependencies
     
 - ### Building from source
-    cd into micro-integrator home directory
+    Navigate into product-mi-tooling home directory
```
[the remark this answers is not in the document]
Comment (sachithKay) [reply cmd/README.md]: fixed

File: monitoring-dashboard/README.md [hunk 9]
```diff
@@ -4,9 +4,9 @@
 
 ### Setting up the development environment
 1. Install Node.js [8.X.X](https://nodejs.org/en/download/releases/).
-2. Fork the [Micro Integrator repository](https://github.com/wso2/micro-integrator).
+2. Fork the [Micro Integrator Tooling repository](https://github.com/wso2/product-mi-tooling).
 3. Clone your fork into any directory.
-4. Access into cloned directory and then navigate into `micro-integrator/tooling/monitoring-dashboard`. This 
+4. Access into cloned directory and then navigate into `product-mi-tooling/monitoring-dashboard`. This
```
[the remark this answers is not in the document]
Comment (sachithKay) [reply monitoring-dashboard/README.md]: fixed

File: cmd/README.md [hunk 10]
```diff
@@ -8,13 +8,13 @@ Command Line tool for managing the WSO2 Micro Integrator
 
 - ### Setting up the development environment
```
[the remark this answers is not in the document]
Comment (sachithKay) [reply cmd/README.md]: fixed

File: cmd/README.md [hunk 11]
```diff
@@ -1,24 +1,24 @@
...
+    3. Clone your fork into any directory.
+    5. Access the cloned directory and then navigate to `product-mi-tooling/cmd`.
+    6. Execute `go mod vendor` or `go mod download` to download all the dependencies.
     
 - ### Building from source
-    cd into micro-integrator home directory
+    Navigate to the product-mi-tooling home directory
 
-    Execute `make install` to build both the **Micro Integrator** and **MIcro Integrator CLI** at once.
+    Execute `make install-cli VERSION=VESION_OF_CHOICE` to build the **MIcro Integrator CLI**.
```
Comment (sachithKay) [inline cmd/README.md]: @menuka94 please confirm this is correct.

File: README.md (hunk 2 above)
Comment (sachithKay) [reply README.md]: fixed

File: cmd/README.md (hunk 11 above)
Comment (menuka94) [reply cmd/README.md]: It is correct.

**Pull request merged**